src/pages/home/report/ReportActionsView.js

```diff
@@ -96,10 +98,13 @@ class ReportActionsView extends React.Component {
 
         this.state = {
             isLoadingMoreChats: false,
+            contextMenuContext: ContextMenuContextDefaultValue,
```

Comment:
I'm honestly not too experienced with the React Context API, but I'm very hesitant to sign off on using it, because:

1. AFAIK we don't use it anywhere else in the app,
2. Honestly, I feel like it leads to very confusing code.

Reply:
Otherwise, we have to move a few callbacks around. Let's get @marcaaron 's input on it.

Reply:
I can't really think of an alternative for the Context API in this case because Onyx would be much less performant.

Reply:
I have no problems with using it if it's valuable, but I'm also not sold on how it's being used here and unsure if it's totally necessary.

When you think about it we only need a central "context" in the first place so that report action items can hide/show the context menus + pass props to them. We can use context for that or we can make those methods available on a lib like `ReportActionContextMenu` and then register the methods in `PopoverReportActionContextMenu`. Or we can try simply passing them as props.

This feels more in line with how we've done things in the past. Also my understanding of a good time to use context is when you have some child component that need to consume _changing data values_ when that context value changes in some way. In this case, we are just passing methods to context and the value of the context never changes so the children never update with the new values. So, it feels like we don't really need to use it at all.